r25-rooms.yml

```diff
@@ -0,0 +1,64 @@
+---

```

Comment:
why are there xml, csv and yml files. i'd suggest either changing this to a csv or consolidate the file types in some way.